src/backend/distributed/shared_library_init.c

```diff
@@ -2305,13 +2316,14 @@ RegisterCitusConfigVariables(void)
 		WarnIfReplicationModelIsSet, NULL, NULL);
 
 	DefineCustomBoolVariable(
-		"citus.running_under_isolation_test",
+		"citus.running_under_citus_test_suite",
 		gettext_noop(
 			"Only useful for testing purposes, when set to true, Citus does some "
-			"tricks to implement useful isolation tests with rebalancing. Should "
+			"tricks to implement useful isolation tests with rebalancing. It also "
+			"registers a dummy label provider for SECURITY LABEL tests. Should "
```

Comment:
Do we have a structure which prevents enabling of a flag in production system? If so, we can arrange this parameter in this way

Reply:
Not that I know of. @JelteF @onurctirtir any idea?

Reply:
Unfortunately we don't have a way to undertand whether it's a production or not. Maybe we could hava used `USE_ASSERT_CHECKING` macro. However, for example, we don't enable asserts in CI so `USE_ASSERT_CHECKING` being false doesn't always mean that it's a production build.

Reply:
I think we need one